Automattic/jetpack
# Add search input for responses inbox and styles

## Proposed changes:
This PR adds a search input and some styles for proper handling the actions row. 
It also adds a _root_ style file, where app wide variables can be set for later reuse

### Other information:


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Checkout and see the new search input. Type and in 500ms (or hit enter) will be fetched.
